services/blockchain-indexer/shared/database/mysqlKVStore.js

```diff
@@ -60,13 +53,43 @@ const get = async (key) => {
 	return value;
 };
 
+const get = async (key) => {
+	const keyValueDB = await getKeyValueStoreIndex();
+
+	const [{ value, type } = {}] = await keyValueDB.find(
+		{ key, limit: 1 },
+		['value', 'type'],
+	);
+
+	return formatValue(value, type);
+};
+
+const getPattern = async (pattern) => {
+	const keyValueDB = await getKeyValueStoreIndex();
+
+	const result = await keyValueDB.find(
+		{ search: { property: 'key', pattern } },
+		['key', 'value', 'type'],
+	);
+
+	result.forEach(row => row.value = formatValue(row.value, row.type));
+	return result;
+};
+
 const deleteEntry = async (key) => {
 	const keyValueDB = await getKeyValueStoreIndex();
 	return keyValueDB.deleteByPrimaryKey([key]);
 };
 
+const KEYS = {
+	TOTAL_LOCKED_PREFIX: 'total_locked',
+	TOTAL_STAKED_PREFIX: 'total_staked',
+};
```

Comment:
Please do not pollute the DB specific implementation with logic from the application.

Reply:
will move to config

Reply:
moved under constants instead.